src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -92,10 +82,10 @@ prov:wasGeneratedBy
   dct:license <https://github.com/DataBiosphere/terra-interoperability-model/blob/master/LICENSE> ;
   rdfs:comment "Please cite The Broad Institute of Harvard and MIT, Data Sciences Platform." ;
   owl:imports <http://www.w3.org/2002/07/owl> ;
-  owl:imports <https://datamodel.terra.bio/EFO_subset> ;
-  owl:imports <https://datamodel.terra.bio/NCBITaxon_Organisms_subset> ;
-  owl:imports <https://datamodel.terra.bio/OBIAssay_subset> ;
-  owl:imports <https://datamodel.terra.bio/OBI_core> ;
+  owl:imports <https://datamodel.terra.bio/imports/EFO_subset> ;
+  owl:imports <https://datamodel.terra.bio/imports/NCBITaxon_Organisms_subset> ;
+  owl:imports <https://datamodel.terra.bio/imports/OBIAssay_subset> ;
+  owl:imports <https://datamodel.terra.bio/imports/OBI_core> ;
   owl:imports <https://datamodel.terra.bio/TerraCoreValueSets> ;
   owl:imports <https://datamodel.terra.bio/TerraDCAT_ap> ;
```

Comment:
Should this be TerraDCAT-AP ?

Reply:
No, this link is valid.  We may want to consider renaming the file to match the ontology name, but the URL matches the ontology name.